components/math/multiplicationConnect/Modal.tsx

```diff
@@ -8,6 +18,12 @@ interface ModalProps {
 
 const Modal: FC<ModalProps> = ({ type, closeModal, children }) => {
   const ref = useRef<HTMLDivElement>();
+  const { newGame } = useSelector(multiplicationConnectSelector);
+  const dispatch = useDispatch();
+  const { user } = useAuth();
+  const [updateUserWin] = useMutation(UPDATE_USER_WIN_MCDATA);
+  // fixme: to be executed after game loss & alert modal close
+  const [updateUserLoss] = useMutation(UPDATE_USER_LOSS_MCDATA);
```

Comment:
is this mutation being used right now? best practice is not to include it in your PRs, unless you're going to use it.